packages/transition-frontend/webpack.config.js

```diff
@@ -30,9 +30,15 @@ const entryFileName =  './lib/app-transition.js';
 
 module.exports = (env) => {
   console.log(`building js for project ${config.projectShortname}`);
+  console.log('process.env.NODE_ENV', process.env.NODE_ENV);
+
+  // Converted from webpack 4
```

Comment:
Is there something more we should be doing with Webpack 5? This converted from webpack feels like we are doing something wrong.